pkg/controller/template/test_data/templates/master/01-master-kubelet/aws/units/kubelet.service

```diff
@@ -0,0 +1,35 @@
+contents: |
+  [Unit]
+  Description=Kubernetes Kubelet
+  Wants=rpc-statd.service
+
+  [Service]
+  Type=notify
+  ExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests
+  EnvironmentFile=-/etc/kubernetes/kubelet-workaround
+  EnvironmentFile=-/etc/kubernetes/kubelet-env
+
+  ExecStart=/usr/bin/hyperkube \
+      kubelet \
+        --config=/etc/kubernetes/kubelet.conf \
+        --bootstrap-kubeconfig=/etc/kubernetes/kubeconfig \
+        --rotate-certificates \
+        --kubeconfig=/var/lib/kubelet/kubeconfig \
+        --container-runtime=remote \
+        --container-runtime-endpoint=/var/run/crio/crio.sock \
+        --allow-privileged \
+        --node-labels=node-role.kubernetes.io/master \
+        --minimum-container-ttl-duration=6m0s \
+        --client-ca-file=/etc/kubernetes/ca.crt \
+        --cloud-provider=aws \
+        \
```

Comment:
Why this empty line?  (spun off from [here][1])

[1]: https://github.com/openshift/installer/pull/1331#discussion_r261401041